composer.json

```diff
@@ -65,6 +65,7 @@
         "prestashop/gridhtml": "^2",
         "prestashop/gsitemap": "^4",
         "prestashop/pagesnotfound": "^2",
+        "prestashop/productcomments": "^5.0",
```

Comment:
I don't understand why this appears as a change because it is simply the same as https://github.com/PrestaShop/PrestaShop/commit/b3e164cb5b1fb232bb1a2e34a75e2474d71800ee

Reply:
Because `productcomments` has been removed and added (yesterday) on `develop`